addons/sale_project/models/sale_order_line.py

```diff
@@ -178,7 +178,7 @@ def _timesheet_create_project_prepare_values(self):
             'active': True,
             'company_id': self.company_id.id,
             'allow_billable': True,
-            'user_id': False,
+            'user_id': self.product_id.project_template_id.user_id.id,
```

Comment:
In some cases, the product_id has no template set, yet the creation of a new project is still triggered. I'd suggest doing this instead
```py
'user_id': self.product_id.project_template_id.user_id.id if self.product_id.project_template_id else False
```
to avoid traceback errors.

Reply:
Hello @damrOdoo 
When product_id has no template set, it will return the false value, so IMO the traceback will not occur.

Reply:
indeed, you're right. Too much JS ain't good for my brain, i mixed up the language :D